src/Ray.BiliBiliTool.Config/Options/DailyTaskOptions.cs

```diff
@@ -77,6 +77,15 @@ public string ChargeComment
         /// </summary>
         public string DevicePlatform { get; set; } = "android";
 
+        /// <summary>
+        /// 自定义漫画阅读 comic_id
+        /// </summary>
+        public int CustomComicId { get; set; } = 27355;
```

Comment:
May be we can change int to long

Reply:
I think that int is sufficient, but of course I can also use long if you insist on it.